app.rb

```diff
@@ -0,0 +1,116 @@
+require_relative './book'
+require_relative './person'
+require_relative './rental'
+require_relative './classroom'
+require_relative './student'
+require_relative './teacher'
+
+# app class
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def start_program
+    puts 'Welcome to my School Library!'
+    until options_list
+      input = gets.chomp
+      if input == '7'
+        puts 'Bye bye :)'
+        break
+      end
+
+      option input
+    end
+  end
+
+  def list_all_books
+    puts 'No book found, try adding some books first.' if @books.empty?
+    @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def list_all_people
+    puts 'We didn\'t find any, please add people first and try again.' if @people.empty?
+    @people.each { |person| puts "Name: #{person.name}, Age: #{person.age}" }
+  end
+
+  def create_person
+    puts 'If you want to create a student insert (1) otherwise insert (2) to create a teacher?'
+    selection = gets.chomp
+
+    case selection
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    end
```

Comment:
- [ ] Let's handle the case where a user enters a different value from 1 or 2 to prevent a potential bug.